src/server/jobAds/ConfigVars.ts

```diff
@@ -0,0 +1,19 @@
+import { articleRealtionsIncluder } from '@/cms/articles/ConfigVars'
+
+export const jobAdArticleRealtionsIncluder = {
+    article: {
+        include: articleRealtionsIncluder
+    }
+} as const
```

Comment:
For better intellisense you can add "satisfies Prisma.JobAdInclude" after "as const".